scripts/binary/get-published-artifacts.ts

```diff
@@ -0,0 +1,127 @@
+import fs from 'fs-extra'
+// @ts-ignore
+import rp from '@cypress/request-promise'
+import util from 'util'
+import { exec } from 'child_process'
+
+const execPromise = util.promisify(exec)
+
+const artifactJobName = 'publish-binary'
+
+const artifactPaths = [
+  '~/cypress/binary-url.json',
+  '~/cypress/npm-package-url.json',
+  '~/cypress/cypress.zip',
+  '~/cypress/cypress.tgz',
+]
+
+function getRequestOptions (url: string) {
+  return {
+    method: 'GET',
+    url,
+    headers: { 'Circle-Token': process.env.CIRCLE_TOKEN },
+  }
+}
+
+function getPipelineId (pipelineInfoFilePath: string) {
+  const data = fs.readFileSync(pipelineInfoFilePath)
+
+  // @ts-ignore
+  const parsedPipelineId = JSON.parse(data).id
+
+  if (!parsedPipelineId) {
+    throw new Error(`error retrieving pipeline id from ${pipelineInfoFilePath}`)
+  }
+
+  return parsedPipelineId
+}
+
+async function getWorkflows (pipelineId: string) {
+  const response = await rp(getRequestOptions(`https://circleci.com/api/v2/pipeline/${pipelineId}/workflow`))
+
+  const parsed = JSON.parse(response)
+
+  if (parsed.items.length === 0) {
+    throw new Error(`did not find any workflows in pipeline ${pipelineId}`)
+  }
+
+  if (parsed.items.length > 1) {
+    console.log(parsed.items)
```

Comment:
Do we need this log?

Reply:
Not necessarily, I just thought it would be good to see the list if something goes wrong here. First thing I would do if I saw this error would be to add this log, so I figured I'd just add it proactively